traitsui/qt4/ui_panel.py

```diff
@@ -1349,11 +1349,13 @@ def __init__(self, parent, html_content, scale_dx, scale_dy):
 HeadingText = None
 
 
-def heading_text(*args, **kw):
+def heading_text(*args, create=False, **kw):
```

Comment:
should the new `create` kwarg be passed along to the `widget.create` method call?

Reply:
Indeed.  Thank you!